src/qt/forms/configurenamedialog.ui

```diff
@@ -158,43 +160,13 @@
        </item>
       </layout>
      </item>
-     <item row="8" column="1">
+     <item row="7" column="1">
       <widget class="QLabel" name="labelTransferToHint">
        <property name="text">
         <string>(Leave empty if not transferring.)</string>
        </property>
       </widget>
      </item>
-     <item row="4" column="1">
-      <layout class="QHBoxLayout" name="horizontalLayout_3">
-       <item>
-        <widget class="QLineEdit" name="dataEdit">
-         <property name="toolTip">
-          <string>Enter JSON string that will be associated with the name</string>
-         </property>
-         <property name="maxLength">
-          <number>520</number>
-         </property>
-        </widget>
-       </item>
-       <item>
-        <widget class="QLabel" name="dataSize">
-         <property name="text">
-          <string>0 / 520</string>
-         </property>
-        </widget>
-       </item>
-       <!--
-       <item>
-        <widget class="QPushButton" name="btnDNSEditor">
-         <property name="text">
-          <string>DNS Editor&#8230;</string>
-         </property>
-        </widget>
-       </item>
-       -->
```

Comment:
I think Qt Designer ate this XML comment when you changed the layout and moved its parent element to lines 64-84. Can you re-add this comment via manual text editing? (I don't know of a way to make Qt Designer not eat them, yes I know it's annoying.)